tools/src/test/scala/caliban/tools/ClientWriterViewSpec.scala

```diff
@@ -789,7 +789,7 @@ object Client {
       User21Selection,
       User22Selection,
       User23Selection
-    ] = (user1(selectionArgs.user1Selection) ~ user2(selectionArgs.user2Selection) ~ user3(
+    ] = (((user1(selectionArgs.user1Selection) ~ user2(selectionArgs.user2Selection) ~ user3(
```

Comment:
Oh so we had a test for it but that produces code that didn't compile? 😆 That sounds like a good use case to add a scripted test, to make sure the generated code compiles. Can you add this test case to the scripted suite? For example here https://github.com/ghostdogpr/caliban/tree/master/codegen-sbt/src/sbt-test/codegen/test-compile/src/main/graphql/genview

Reply:
Yeah, it was a surprise to me also that there was existing code to handle large types. At the start I wrote a separate test, ran it and it *and* the existing test failed :laughing: . That is how I found out! I tested the changes manually in the project I am using Caliban and it worked.

I have added the type to the codegen-set genview test. See what you think